storage/client_raft_test.go

```diff
@@ -1133,7 +1132,7 @@ func TestRaftAfterRemoveRange(t *testing.T) {
 // number of repetitions adds an unacceptable amount of test runtime).
 func TestRaftRemoveRace(t *testing.T) {
 	defer leaktest.AfterTest(t)
-	t.Skip("TODO(bdarnell): #768")
+	t.Skip("TODO(bdarnell): fix TestRaftRemoveRace")
```

Comment:
When the skip was added, I thought it would be fixed by replica tombstones. That turned out not to be the case, so it's no longer linked to #768.